test/browser_tests/cucumber-new/features/suites/apps/rural_and_underserved_tool.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Verify the Rural and underserved areas tool landing page works according to requirements
+  As a first time visitor to the Rural and underserved areas tool landing page
+  I want to Check status of properties for loans extended on the landing page
+  So that I can find the information I'm looking for
+
+Background:
+  Given I navigate to the Rural and underserved areas tool landing page
+
+@smoke_testing @rural
+Scenario Outline: Check status of properties for loans extended
+  Given I navigate to the Rural and underserved areas tool landing page
+  When I check status of properties for loans extended in <year>
+  Then I Enter addresses manually <address>
+  And I click on the "Check addresses" button
```

Comment:
Missing `Then` here: What are we asserting will happen?

Reply:
Added identified text to be found on rural and underserved page after clicking "Check addresses" button